packages/schema/src/config/dev.ts

```diff
@@ -28,11 +28,11 @@ export default defineUntypedSchema({
     port: process.env.NUXT_PORT || process.env.NITRO_PORT || process.env.PORT || 3000,
 
     /** Dev server listening host */
-    host: process.env.NUXT_HOST || process.env.NITRO_HOST || process.env.HOST || 'localhost',
+    host: process.env.NUXT_HOST || process.env.NITRO_HOST || process.env.HOST || '0.0.0.0',
 
     /**
      * Listening dev server url
```

Comment:
```suggestion
     * Listening dev server URL.
     *
     * This should not be set directly as it will always be overridden by the
     * dev server with the full URL (for module and internal use).
```